### ISSUE TITLE
Implement and Test Generative Image Editing Model

### ISSUE THREAD

**Issue by navv42** [navv42/style-up#2]
This task involves setting up the `Qwen/Qwen-Image-Edit` model to perform a targeted edit on an image. For this initial test, we will use a **manually created mask** to prove the model's capability before integrating it with the output from our segmentation model. Here is the documentation on how to use the model:

```
from diffusers import DiffusionPipeline
from diffusers.utils import load_image

pipe = DiffusionPipeline.from_pretrained("Qwen/Qwen-Image-Edit")

prompt = "Turn this cat into a dog"
input_image = load_image("https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/cat.png")

image = pipe(image=input_image, prompt=prompt).images[0]
```

```
Quick Start

Install the latest version of diffusers

pip install git+https://github.com/huggingface/diffusers

The following contains a code snippet illustrating how to use the model to generate images based on text prompts:

import os
from PIL import Image
import torch

from diffusers import QwenImageEditPipeline

pipeline = QwenImageEditPipeline.from_pretrained("Qwen/Qwen-Image-Edit")
print("pipeline loaded")
pipeline.to(torch.bfloat16)
pipeline.to("cuda")
pipeline.set_progress_bar_config(disable=None)
image = Image.open("./input.png").convert("RGB")
prompt = "Change the rabbit's color to purple, with a flash light background."
inputs = {
    "image": image,
    "prompt": prompt,
    "generator": torch.manual_seed(0),
    "true_cfg_scale": 4.0,
    "negative_prompt": " ",
    "num_inference_steps": 50,
}

with torch.inference_mode():
    output = pipeline(**inputs)
    output_image = output.images[0]
    output_image.save("output_image_edit.png")
    print("image saved at", os.path.abspath("output_image_edit.png"))
```

**Acceptance Criteria:**

1.  A new directory `ml/editing_service` is created.
2.  A Python script `ml/editing_service/edit.py` is created.
3.  The script accepts three command-line arguments:
    - `--original_image`: The path to the source image.
    - `--mask_image`: The path to a black-and-white mask image where the white area indicates the region to be edited.
    - `--prompt`: A text string describing the desired change (e.g., "a red leather jacket").
    - Example: `python edit.py --original_image ../test_images/person1.jpg --mask_image ../test_images/person1_shirt_mask.png --prompt "a red leather jacket"`
4.  The script successfully loads the `Qwen/Qwen-Image-Edit` model. **Note:** Document the hardware requirements (e.g., GPU with >20GB VRAM) in a `README.md` within the directory.
5.  The script generates a new image with the specified edit applied **only to the masked region** and saves it to a file (e.g., `person1_edited.png`).
6.  A `requirements.txt` file is included.
7.  The task is complete when at least one test image is successfully modified according to a prompt, using a manually created mask (e.g., created in GIMP or Photoshop).
